utils/cobrautil/bind.go

```diff
@@ -55,7 +55,11 @@ func BindAll(cmd *cobra.Command, envPrefix, configFileFlagName string) error {
 		ok = true
 		fs.VisitAll(func(f *pflag.Flag) {
 			if !f.Changed && v.IsSet(f.Name) {
-				if err := fs.Set(f.Name, fmt.Sprintf("%v", v.Get(f.Name))); err != nil {
+				s := fmt.Sprintf("%v", v.Get(f.Name))
+				s = strings.TrimPrefix(s, "[")
```

Comment:
Is it because we don't have access to the anyflag repo right now? I think it could wait a few days (unless it's problematic to fix in anyflag)

Reply:
After investigation, it turns out that anyflag follows parsing rules of cobra, so we probably don't want to mess with it. 
More advanced parsing is usually implemented in a higher layer and later passed to cobra like I've done here.